underdocs-renderer/src/main/resources/main.css

```diff
@@ -351,28 +357,36 @@ h2 {
 
 .submodules-table,
 .subheaders-table,
-.elements-table {
+.elements-table,
+.return-value-table {
     border-spacing: 0;
-    width: 75%;
-    max-width: 75%;
+    width: 100%;
+    max-width: 100%;
 }
 
 .submodules-table .submodule-name-cell, 
 .submodules-table .submodule-excerpt-cell,
 .subheaders-table .subheader-name-cell, 
 .subheaders-table .subheader-excerpt-cell,
 .elements-table .element-name-cell, 
-.elements-table .element-excerpt-cell {
+.elements-table .element-excerpt-cell,
+.return-value-table .return-value-name-cell,
+.return-value-title .return-value-excerpt-cell {
     border-top: var(--table-border);
     padding-top: 0.3rem;
     padding-bottom: 0.3rem;
 }
 
+.return-value-table tr:nth-child(even) {
+    background-color: var(--table-row-background);
+}
```

Comment:
Although a nice addition, alternating colors are not used throughout, so in my opinion, we should not diverge from that choice here either.

Reply:
Alright